src/remote-config/condition-evaluator-internal.ts

```diff
@@ -26,6 +26,7 @@ import {
   PercentConditionOperator
 } from './remote-config-api';
 import * as farmhash from 'farmhash';
+import long = require('long');
```

Comment:
Chatted a bit with @erikeldridge  offline and we're not sure what the preference is. @lahirumaramba do you have thoughts on this?

Reply:
I am guessing this is due to the package `long` switching to ESM since `v5`. I think we are okay with using `import long = require('long')` or `const { long } = require('long')` here. Thank you!

Reply:
Sounds good, I'll leave this as is then. Thanks Lahiru!